src/main/java/ledger/user_interface/ui_controllers/MainPageController.java

```diff
@@ -150,7 +150,8 @@ private void createAccountPopup() {
      */
     private void deleteAccount() {
         if (chooseAccount.getSelectedAccount() == null) {
-            setupErrorPopup("Cannot delete the All Accounts aggregation", new Exception());
+            setupErrorPopup("Cannot delete the All Accounts aggregation. Please select an account " +
+                    "to delete.", new Exception());
```

Comment:
Thats just the way we've always done it. The exception is never thrown, so the overhead is negligible. I can make another one if oyu like.